src/System.Private.CoreLib/shared/System/Collections/Generic/List.cs

```diff
@@ -23,11 +23,11 @@ public class List<T> : IList<T>, IList, IReadOnlyList<T>
     {
         private const int DefaultCapacity = 4;
 
-        private T[] _items; // Do not rename (binary serialization)
+        private InvariantArray<T> _items; // Do not rename (binary serialization)
```

Comment:
This is binary serialization breaking change

Reply:
(ok, I see that you have mentioned it in the description)

Reply:
Yup. See _open issues_ in the PR for discussion on ways to mitigate this.